addons/binding/org.openhab.binding.atlona/README.md

```diff
@@ -0,0 +1,339 @@
+# Atlona Binding
+
+This binding integrates Atlona AT-UHD-PRO3 HdBaseT matrix switches (http://www.atlona.com) into your openHAB installation.
```

Comment:
make it a link: `[Atlona AT-UHD-PRO3 HdBaseT matrix switches](http://www.atlona.com)`